<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>智慧停车系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入6-8位帐号!</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>帐号：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入6-10位密码!</string>
   </property>
  </widget>
  <widget class="QLabel" name="code">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>验证码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="code_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_butt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_butt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eye">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
